src/hooks/auth0-provider.tsx

```diff
@@ -105,10 +117,12 @@ const Auth0Provider = ({domain, clientId, children}) => {
   );
 
   const getCredentials = useCallback(
-    async (...options) => {
+    async (...options: any) => {
```

Comment:
Would it make sense to provide a typed options parameter here instead of resorting to any?